.circleci/config.yml

```diff
@@ -10,6 +10,7 @@ orbs:
   coveralls: coveralls/coveralls@1.0.6
 jobs:
   test-mad-dog-e2e:
+    parallelism: 10
```

Comment:
what's the purpose of bumping this to 10 parallel tests?

Reply:
i'm gonna revert it, just for testing my changes more quickly

Reply:
are we going to leave this parallelism setting here for merge?

Reply:
reverted